infra/database.yaml

```diff
@@ -1,5 +1,5 @@
 AWSTemplateFormatVersion: '2010-09-09'
-Description: ''
+Description: 'The database - including the networking required to get it to interface with the VPC'
```

Comment:
suggestion: I am not sure what "VPC" means, mind expanding the acronym?

Reply:
Sure!  It's amazon's word for "network".  Stands for "virtual private cloud", by analogy to VPN.  It's just a buncha private IPs, subnets and route tables.  Replaced with "the private network".